supports-color.js

```diff
@@ -0,0 +1,25 @@
+"use strict";
```

Comment:
I think it should be moved to `lib` (as it's more an internal util and not part of public API)